includes/Story_Post_Type.php

```diff
@@ -140,17 +139,6 @@ public function register() {
 		add_filter( 'bulk_post_updated_messages', [ $this, 'bulk_post_updated_messages' ], 10, 2 );
 	}
 
-	/**
-	 * Deactivate the service.
-	 *
-	 * @param bool $network_wide Whether the deactivation was done network-wide.
-	 * @return void
-	 */
-	public function deactivate( $network_wide ) {
-		unregister_post_type( self::POST_TYPE_SLUG );
```

Comment:
Sure about that?

I thought we need to remove the post type upon plugin deactivation before flushing rewrite rules, otherwise the web-story permalinks would not be flushed.

Reply:
I put it back here - https://github.com/google/web-stories-wp/pull/7503/commits/895e449deaa929ea681f0df039873f665fea9163